proj/src/main/webapp/chat.js

```diff
@@ -421,7 +421,7 @@ function pushChatMessage() {
         }
         // push message to datastore
         let dbRefObj = getActiveDbRef();
-        const messageKey = dbRefObj.push(message).key;
+        dbRefObj.push(message).key;
```

Comment:
do you need the `.key` at the end still?

Reply:
nope. fixed